src/main/java/seedu/stockpal/commands/HistoryCommand.java

```diff
@@ -33,8 +33,8 @@ public HistoryCommand(Integer pidValue) {
 
     @Override
     public void execute(ProductList productList, TransactionList transactionList) throws StockPalException {
-        int productIndex = productList.findProductIndex(pid);
-        TransactionList.findTransactions(transactionList, productIndex + 1);
+        productList.findProductIndex(pid);
```

Comment:
Would this line still be necessary?

Reply:
I would have to check whether PID is valid. Even though I do not need the index, this line allows me to check PID validity.